src/braket/circuits/gates.py

```diff
@@ -976,22 +1163,28 @@ def fixed_qubit_count() -> int:
     @staticmethod
     @circuit.subroutine(register=True)
     def iswap(
-        target1: QubitInput, target2: QubitInput, *, control: Optional[QubitSetInput] = None
+        target1: QubitInput,
+        target2: QubitInput,
+        *,
+        control: Optional[QubitSetInput] = None,
+        control_state: Optional[BasisStateInput] = None,
     ) -> Instruction:
         """Registers this function into the circuit class.
 
         Args:
             target1 (QubitInput): Target qubit 1 index.
             target2 (QubitInput): Target qubit 2 index.
-            control (Optional[QubitSetInput]): Control qubit(s).
+            control (Optional[QubitSetInput]): Control qubit(s). Default None.
```

Comment:
Needs `control_state` docstring
```suggestion
            control (Optional[QubitSetInput]): Control qubit(s). Default None.
            control_state (Optional[BasisStateInput]): Quantum state on which to control the
                operation. Must be a binary sequence of same length as number of qubits in
                `control`. Will be ignored if `control` is not present. May be represented as a
                string, list, or int. For example "0101", [0, 1, 0, 1], 5 all represent
                controlling on qubits 0 and 2 being in the |0⟩ state and qubits 1 and 3 being
                in the |1⟩ state. Default "1" * len(control).
```